content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md

```diff
@@ -119,7 +130,11 @@ option. Your cluster requirements may need a different configuration.
 {{< note >}}
 Some CNI network plugins like Calico require a CIDR such as `192.168.0.0/16` and
 some like Weave do not. See the [CNI network
-documentation](/docs/setup/independent/create-cluster-kubeadm/#pod-network).
+<<<<<<< HEAD:content/en/docs/setup/production-environment/tools/independent/high-availability.md
```

Comment:
This looks like a merge left in by mistake.

Reply:
Thank you it was a miss. Fixed it now.